app/uk/gov/hmrc/gform/ApplicationLoader.scala

```diff
@@ -272,8 +272,6 @@ class ApplicationModule(context: Context)
   )
 
   override lazy val csrfErrorHandler: CSRF.ErrorHandler = new CSRF.CSRFHttpErrorHandler(csrfHttpErrorHandler)
-  override lazy val httpRequestHandler: HttpRequestHandler = routingModule.httpRequestHandler
```

Comment:
Can we remove all `CustomHttpRequestHandler` if it is not needed? 

Has this been thoroughly tested? I'm not sue why that dropping of trailing `/`  is there for.

Reply:
I reverted this change because it will cause some problems.